projects/js-packages/components/components/contextual-upgrade-trigger/index.tsx

```diff
@@ -1,18 +1,31 @@
 import { Icon, arrowRight } from '@wordpress/icons';
+import classnames from 'classnames';
 import Text from '../text';
 import styles from './style.module.scss';
 import { CutBaseProps } from './types';
 import type React from 'react';
 
-const ContextualUpgradeTrigger: React.FC< CutBaseProps > = ( { description, cta, onClick } ) => {
+const ContextualUpgradeTrigger: React.FC< CutBaseProps > = ( {
+	description,
+	cta,
+	onClick,
+	href,
+	target,
```

Comment:
I don't think we need to open for `target`, let's fix it as `_blank` and we're good. Don't think a case that we want to use something different than `_blank`.

Reply:
Sounds good! Fixed in [41e928d](https://github.com/Automattic/jetpack/pull/26115/commits/41e928d18fb4e7cd5ad4f45301416494370a8b80) 👍🏼